README.md

```diff
@@ -167,9 +167,19 @@ So, for example, to get my [git deck](https://github.com/Stvad/Software_Engineer
 ## Snapshots
 
 **CrowdAnki** can help you preserve **the history of edits for your decks**.  
-It does this by exporting them in a specified location and creating a git commit each time you do a snapshot.
-
-You can take snapshots manually via `File > CrowdAnki: Snapshot` menu action.  
-Or you can enable automated snapshots in plugin configuration. You can find more details on how to do it [here](crowd_anki/config.md) 
-
-Other things you can [configure](crowd_anki/config.md) is the location of the snapshot and what decks should be included into it. 
+It does this by exporting them in a specified location and creating a git commit each time you do a snapshot. 
+
+You can take snapshots manually via `File > CrowdAnki: Snapshot` menu action. 
+Or you can enable automated snapshots in add-on configuration (see below) which will take a snapshot each time you open or close anki.
+
+## Configuration Settings
+You can find the add-on's config page via `Tools -> Add-ons, select CrowdAnki then hit Config`. 
+In here you can set a some config options, as well as a detailed explanation on what each does, with examples and default behaviour.
+
+Here is a general summary of the currently available config options:
+ - `Snapshot Path`: location of the snapshot will be exported to
+ - `Snapshot Root Decks`: what decks should be included in the snapshot
```

Comment:
's not quite that (https://github.com/Stvad/CrowdAnki/blob/585f570b3ebcd06ee3500caf661290f95cc0943e/crowd_anki/config.md)

Reply:
Fixed :+1: